app/controllers/criteria_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CriteriaController < ApplicationController
+  def new
+    @criterium = Criterium.new
```

Comment:
It could be done either way. this way we could load the view template and call the component from in the vue, or we could do it with only a vue component. i prefer the latter, in which case this wouldn't be needed, but yeah: i'll leave because it doesn't hurt and we might end up loading the view